### PR TITLE
Bundle BKVM.org - BookKeeper Visual Manager 1.2.0

### PR DESCRIPTION
**Pull request by eolivelli**
- add dependency to BKVM 1.2.0 war
- make explicit a few dependencies
- start bkvm.org on backend service at path http://localhost:8080/bkvm (user: admin by default)
- BKVM is a Java War application, we are deploying it using the Embedded Tomcat that comes with SpringBoot
- added an example bkvm.conf file
- by default BKVM is disabled, you have to change **bkvm.enabled** to **true** and configure the connection to your BK cluster
- by default BKVM will use the embedded HerdDB database deployed inside the Pulsar Manager Backend Process (there is no support for PostGreSQL in BKVM)
- BKVM 1.x supports a single BK cluster, support for multiple environments will come in BKVM 2.x

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -13,7 +13,7 @@
 #
 
 spring.cloud.refresh.refreshable=none
-server.port=7750
+server.port=8080
```
Comment (eolivelli) [inline src/main/resources/application.properties:16]: This change is needed for running Pulsar Manager only using the Java Backend Service.
But we can leave 7750 by now and have a configuration guide

Comment (tuteng) [reply src/main/resources/application.properties:16]: Shall we consider changing a port? The port 8080 is easy to cause port conflict?

Comment (eolivelli) [reply src/main/resources/application.properties:16]: To me another port will be okay.
Please check that port 8080 is required below at zuul.routes.admin.url

I suggest using 8888.
I don't know the amount of changes needed to the docs and docker files

if we move forward with bundling the front-end with the back-end than docker files will be far easier (no need for nginx)

**Comment by eolivelli** [comment]
@tuteng I will try to finish this work by the end of the week

@mino181295 we need a release of BKVM

**Comment by eolivelli** [comment]
@tuteng  Travis will pass once BK 1.2.0 is Available to Maven Central mirrors

**Comment by eolivelli** [comment]
@rdhabalia  @sijie  @tuteng patch is ready for review

**Pull request merged**